posts/templatetags/__init__.py

```diff
@@ -0,0 +1 @@
+__author__ = 'juancabello'
```

Comment:
I think this file should be completely empty. I think PyCharm created that file.